container-engine-lib/lib/backend_impls/kubernetes/kubernetes_kurtosis_backend/user_services_functions/destroy_user_services.go

```diff
@@ -37,13 +37,25 @@ func DestroyUserServices(
 	for serviceUuid, serviceObjsAndResources := range allObjectsAndResources {
 		resources := serviceObjsAndResources.KubernetesResources
 
-		pod := resources.Pod
-		if pod != nil {
-			if err := kubernetesManager.RemovePod(ctx, pod); err != nil {
+		serviceToRemove := resources.Service
```

Comment:
This should have been done earlier.  The k8s service needs to be removed in addition to the pod.